gcp/api/server.py

```diff
@@ -301,13 +308,16 @@ def _clean_purl(purl):
 @ndb.tasklet
 def query_by_commit(commit, to_response=bug_to_response):
   """Query by commit."""
-  query = osv.AffectedCommit.query(osv.AffectedCommit.commit == commit,
-                                   osv.AffectedCommit.public == True)  # pylint: disable=singleton-comparison
+  query = osv.AffectedCommits.query(osv.AffectedCommits.commits == commit)
```

Comment:
Is it worth automatically verifying with or falling back to the legacy `AffectedCommit` here, or will that cause too much latency?

Reply:
Good point. It adds a bit of complexity and I'm a bit wary of latency here. I'd like to validate that these indexes are complete a different way.